eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/alert/AlertCommand.java

```diff
@@ -0,0 +1,99 @@
+package com.eternalcode.core.feature.essentials.alert;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.notice.NoticeTextType;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.context.Sender;
+import dev.rollczi.litecommands.annotations.join.Join;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.literal.Literal;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import dev.rollczi.litecommands.annotations.command.Command;
+import org.bukkit.entity.Player;
+
+@Command(name = "alert", aliases = { "broadcast", "bc" })
```

Comment:
I think we should split this like as `TellRawQueueCommand` to separate "alert-queue" command. (also with eternalcore.alert.queue permission)